<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuDialog</class>
 <widget class="QDialog" name="MenuDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Urszula Chmielewska - Shopping List</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>530</width>
     <height>492</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Gill Sans MT&quot;;
color: #FFC700;
font-size: 33px;</string>
      </property>
      <property name="text">
       <string>Shopping List Generator</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">font: 8pt &quot;Gill Sans MT&quot;;
font-size: 12px;
font: bold;
color: white;</string>
              </property>
              <property name="text">
               <string>Your recipes</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="availableRecipes">
              <property name="styleSheet">
               <string notr="true">font: &quot;Gill Sans MT&quot;;
font-size: 11px;
border-style: solid;
border-width: 0.5px;
border-color: #54565B;
background-color: #FFFFFF</string>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="btnAdd">
              <property name="styleSheet">
               <string notr="true">background-color: #FFCB05;
color: white;
font: &quot;Gill Sans MT&quot;;
font-size: 13px;
</string>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnRemove">
              <property name="styleSheet">
               <string notr="true">font: &quot;Gill Sans MT&quot;;
font-size: 13px;
background-color: #FF665A;
color: #FFFFFF;
</string>
              </property>
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">font: 8pt &quot;Gill Sans MT&quot;;
font-size: 12px;
font: bold;
color: white;</string>
            </property>
            <property name="text">
             <string>Chosen recipes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="chosenRecipes">
            <property name="styleSheet">
             <string notr="true">font: &quot;Gill Sans MT&quot;;
font-size: 11px;
border-style: solid;
border-width: 0.5px;
border-color: #54565B;
background-color: #FFFFFF</string>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;Gill Sans MT&quot;;
font-size: 12px;
font: bold;
color: white;</string>
          </property>
          <property name="text">
           <string>Shopping List</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="shopList">
          <property name="styleSheet">
           <string notr="true">font: &quot;Gill Sans MT&quot;;
font-size: 11px;
border-style: solid;
border-width: 0.5px;
border-color: #54565B;
background-color: #FFFFFF</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
